### DISCUSSION TITLE
Battery Life Report

### DISCUSSION THREAD

**Discussion by jburker** [joeycastillo/Sensor-Watch#306] [General]
The battery on my Casio A158W just died after a year’s use: not too bad! Didn’t use the backlight, temperature sensor installed. Ordered a new battery and will replace tomorrow. Thanks for the cool hardware.

**Comment by 814d3** [comment]
Like with any other primary cell there could be a significantly difference in capacity. Consider bying a qualitiy brand. Mine reaches a year soon with a repeated use of the backlight and buzzer. Nevertheless one year is still good I think.

**Comment by jburker** [comment]
Oh, for sure! Not criticizing the battery life, just noting for others.
